app/browser/tabs.js

```diff
@@ -1359,7 +1359,28 @@ const api = {
     return state
   },
   forgetTab: (tabId) => {
-    webContentsCache.cleanupWebContents(tabId)
+    const tab = webContentsCache.getWebContents(tabId)
+    if (!tab) {
+      return
```

Comment:
Should this codepath also call `webContentsCache.cleanupWebContents(tabId)`?

Reply:
I suppose it is conceivable that in `webContentsCache` module, `tabData.tab` could be null, but `tabData` could be an object. I don't _think_ there is a code path where that can happen at the moment, but it wouldn't be bad to explicitly call `cleanupWebContents` here, in case that state happens in the future. wdyt?